style.css

```diff
@@ -349,3 +351,69 @@ a .btn-two {
 .text {
   margin-left: 2vw;
 }
+
+.contact-section {
+  background: #fff;
+}
+
+.contact-form {
+  margin-top: 90px;
+  flex-direction: column;
+  justify-content: start;
+  background-image: url(images/contact-b.png);
+  background-size: contain;
+  border: 1px solid white;
+  border-top-left-radius: 110px;
+  text-align: center;
+}
```

Comment:
- [ ] Kindly work on your style again to fit the [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).
Your design:
![formstyle](https://user-images.githubusercontent.com/55596970/154150687-bb75cfd0-bb2e-4fb7-b50d-fe25ac92e74d.png)
Figma design, please follow it closely 
![figma](https://user-images.githubusercontent.com/55596970/154150835-1801fee6-d479-4d3b-b800-283920f24ba7.png)

Reply:
I haven't done this project yet. Why am I receiving required changes for project I haven't create on my Dashboard ?

Reply:
Sorry @dfizzkruah93  I mistaken mention your name in the review

Reply:
Hello @CollinsTatang thank you for the feedback it has pushed me to researching and learn more. However, i have noticed that the form is stable in some Mobile views and others not especially those that go beyond the recommended width. 
Am writing to you to share your opinion over this problem. Either through helping to it or anything that will work for you.

thank you being a hero . you can reach on slack as misheck livingif or faster replies.

happy coding